pandas/tests/sparse/frame/test_apply.py

```diff
@@ -90,3 +90,22 @@ def test_applymap(frame):
     # just test that it works
     result = frame.applymap(lambda x: x * 2)
     assert isinstance(result, SparseDataFrame)
+
```

Comment:
is there a reason you are putting them in the sparse tests?

these look quite duplicative of the current apply tests, which there are many.

Reply:
@jreback well, there is no specific reason for putting them in sparse tests, as for duplicative tests, I initially committed only one test which threw error until it was recommended to add more

Reply:
@uds5501 : I was saying that you should add each example, but you didn't need to create separate tests with duplicate logic to do so.

Reply:
@gfyoung  Kindly suggest a method, I mean I am little confused here by what adding each example exactly means

Reply:
@uds5501 : I think you actually figured out 😄